test/rockpaperscissors.js

```diff
@@ -0,0 +1,868 @@
+const truffleAssert = require('truffle-assertions');
+const timeMachine = require('ganache-time-traveler');
+const Game = artifacts.require("./RockPaperScissors.sol");
+
+contract('rcp', async accounts => {
+
+    const { toBN, soliditySha3 } = web3.utils;
+    const { getBalance, getTransaction } = web3.eth;
+    const playerOneSecretString = "secret1";
+    const playerTwoSecretString = "secret2";
+    const wrongSecretString = "wrongSecret";
+    
+    const ROCK = 1;
+    const PAPER = 2;
+    const SCISSORS = 3;
+
+    const DRAW = "0";
+    const LEFTWIN = "1";
+    const RIGHTWIN = "2";
+    const INCORRECT = "3";
+
+    const UNSTARTED = "0";
+    const PLAYER_ONE_MOVE = "1";
+    const PLAYER_TWO_MOVE = "2";
+    const PLAYER_ONE_REVEAL = "3";
+
+    const INVALID_MOVE_MSG = "Invalid move";
+    const INVALID_STEP_MSG = "Invalid step";
+    const HASH_MISMATCH_MSG = "Move and secret do not match";
+    const GAME_NOT_EXPIRED_MSG = "Game has not expired";
+    const SECRET_EMPTY_MSG = "Secret is empty";
+    const INVALID_PLAYER_MSG = "Invalid player";
+
+    const ZERO_ADDRESS = "0x0000000000000000000000000000000000000000";
+    const ZERO_BYTES_32 = "0x0000000000000000000000000000000000000000000000000000000000000000";
```

Comment:
Same but fancier: `"0x".padEnd(66, "0");`

Reply:
Thanks for the tip. Looks neater.